test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
@erikvold I'm not sure I'm following you. If we remove all the `widget` from SDK, even the mapping – that I've done now – what we should test? I mean, requiring the `widget` (or `sdk/widget`) module, with this patch, is basically like requiring a `foo` module: it doesn't exist. So what should be the purpose of the test? You're suggesting to test that every module we remove it's actually failing in requiring them? We never did that in the past, and I don't see the advantage, just adding reference to non-existing modules… and moreover, where we should add those tests?
If I didn't understand what you mean, please provide the code you have in mind. Thanks!